jenkins/run

```diff
@@ -55,11 +55,12 @@ if [[ -z "${1}" ]]; then
     exit 1
 fi
 
-_branch_name="$(cat jenkins/branch.name)"
-
+_branch_name=$(cat jenkins/branch.name)
 _image_repo=${IMAGE_REPO:-quay.io/pbench}
-_image_role=${IMAGE_ROLE:-devel}
-_image_kind=${IMAGE_KIND:-fedora}
+_image_name=${IMAGE_NAME:-pbench-ci-fedora}
+_image=${IMAGE:-${_image_repo}/${_image_name}:${_branch_name}}
+
+EXTRA_PODMAN_SWITCHES=${EXTRA_PODMAN_SWITCHES:-"-it"}
```

Comment:
Why are we defaulting to `-it` now, when we didn't before?

Reply:
When the CI uses this script, it specifies a value for `EXTRA_PODMAN_SWITCHES`, so this change has no effect, there.  For human use, I almost always want `-it`, so I made it the default so that I didn't have to provide it explicitly every time.

In terms of conflict, I don't think it actually hurts to have either `-i` or `-t` when you don't need them, but it sure is inconvenient to have them omitted when you _do_ want them....